src/Control/Inciter/Types.h

```diff
@@ -104,6 +104,18 @@ using CompFlowPDEParameters = tk::tuple::tagged_tuple<
   //! Parameter vector (for specific, e.g., verification, problems)
   tag::beta, std::vector< kw::pde_beta::info::expect::type >,
   //! Parameter vector (for specific, e.g., verification, problems)
+  tag::betax, std::vector< kw::pde_betax::info::expect::type >,
+  //! Parameter vector (for specific, e.g., verification, problems)
```

Comment:
Yeah. These are documented (in more or less detail in Control/Keywords.h already). More documentation would of course be good. I will update the documentation for this and the other keywords for these test problems when I will actually test them + add regression tests as well.